src/components/Dashboard/FilingHistoryList/HeaderActions.vue

```diff
@@ -142,6 +143,7 @@ export default class HeaderActions extends Mixins(AllowableActionsMixin) {
     // see also TodoList.vue:doResumeFiling()
     switch (filing?.name) {
       case FilingTypes.ALTERATION:
+      case FilingTypes.AMALGAMATION_APPLICATION:
```

Comment:
This is to redirect to Edit UI for a correction.

Without this, the default was used (line 175), which routes to a local correction notation.